### PR TITLE
Changed values of sound decaying

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

i changed the distance the sound will decay
## Changes

i changed the following volume values
2 to 1 2x
1.7 to 1
5 to 1 3x
## Testability

- [ x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

**Comment by WalshyDev** [comment]
Here are all occurances just in case you wanted quick access since you aren't using an editor...
Values

File: src/me/mrCookieSlime/Slimefun/Objects/tasks/MagnetTask.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ void executeTask() {
 		for (Entity item: Bukkit.getPlayer(uuid).getNearbyEntities(6D, 6D, 6D)) {
 			if (item instanceof Item && !item.hasMetadata("no_pickup") && ((Item) item).getPickupDelay() <= 0) {
 				item.teleport(Bukkit.getPlayer(uuid).getEyeLocation());
-				Bukkit.getPlayer(uuid).getWorld().playSound(Bukkit.getPlayer(uuid).getEyeLocation(), Sound.ENTITY_ENDERMAN_TELEPORT, 5L, 2L);
+				Bukkit.getPlayer(uuid).getWorld().playSound(Bukkit.getPlayer(uuid).getEyeLocation(), Sound.ENTITY_ENDERMAN_TELEPORT, 1L, 2L);
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/Objects/tasks/MagnetTask.java]: Should be F, not L.

Comment (J3fftw1) [reply src/me/mrCookieSlime/Slimefun/Objects/tasks/MagnetTask.java]: it was already like that didnt know why

Comment (WalshyDev) [reply src/me/mrCookieSlime/Slimefun/Objects/tasks/MagnetTask.java]: Also change the other param to F too

**Comment by TheBusyBiscuit** [comment]
Well what about the Infused Hopper and all the other stuff from before...?

**Comment by J3fftw1** [comment]
what do you mean Biscuit

**Comment by TheBusyBiscuit** [comment]
I mean what I said.
You originally wanted to change the Infused Hopper, but you didn't do.
This Pull Requests only contains like 10% of the stuff you wanted to change.

**Comment by J3fftw1** [comment]
changed it right now i think thats all

**Pull request merged**